tests/test_optimizer_random_dag.py

```diff
@@ -98,10 +110,13 @@ def _optimize_by_brute_force(tasks, plan):
             else:
                 objective = _optimize_by_brute_force(tasks[1:], plan)
             if objective < min_objective:
+                final_plan[task] = resources
                 min_objective = objective
         return min_objective
 
-    return _optimize_by_brute_force(topo_order, {})
+    min_objective = _optimize_by_brute_force(topo_order, {})
+    print(final_plan)
```

Comment:
Is this intended? Seems like the `final_plan` variable is only used for debugging. I'm wondering if we could delete it.

Reply:
Let's keep it for future use of debugging. It does not matter to keep it, as this is the test file and won't affect the normal users.